dgraphtest/image.go

```diff
@@ -24,6 +24,7 @@ import (
 	"os/exec"
 	"path/filepath"
 
+	"github.com/dgraph-io/dgraph/testutil"
```

Comment:
fix imports

Reply:
Sorry, I could not quite get this. If you are trying to point out this import is redundant and I should be deleting this line, I added 'testutil.GeneratePlugins(race)' at line #54 in this file hence, I added this import.